sample-status-updater-application/src/main/java/life/qbic/samplestatusupdater/app/SampleUpdater.java

```diff
@@ -72,8 +74,16 @@ public void execute() {
         SampleTrackingService sampleTrackingService = new SampleTrackingServiceConnector(serviceList.get(0), credentials);
 
         OpenBisSearchService searchService = new SampleSearchConnector(session.getApi(), session.getToken());
-        Date lastSearchDate = parseDateFromStringWithPattern((String) properties.get("lastSearchDate"), "yyyy-MM-dd");
+        Instant lastSearchDate = null;
```

Comment:
What do you think about renaming `lastSeachDate` so something like `latestSearchTimepoint` as it is now a point in time rather than a date only.